docs/filters.md

```diff
@@ -220,7 +222,19 @@ Matches issues which were created on a given date, or within a given date range.
 
 *Expects a number or  number range*
 
-Matches issues which were updated in the given number of hours. For example, `updated:<24` would match issues updated in the last day. If a number `n` is given, it is implicitly translated to `<n`.
+Matches issues which were updated in the given number of hours. For example, `updated:<24` would match issues updated in the last day. If a number `n` is given, it is implicitly translated to `<n`. Number ranges are written using a relational operator (.e.g `>5`, `<=10`).
+
+### updated-others
+
+*Expects a number or  number range*
+
+Matches issues which were updated in the given number of hours by users other than the logged in user. For example, `updated:<24` would match issues updated in the last day by other users. Number ranges are written using a relational operator (.e.g `>5`, `<=10`). If a number `n` is given, it is implicitly translated to `<n`.
+
+### updated-self
+
+*Expects a number or  number range*
+
+Matches issues which were updated in the given number of hours by the logged in user. For example, `updated:<24` would match issues updated in the last day by the logged in user. Number ranges are written using a relational operator (.e.g `>5`, `<=10`). If a number `n` is given, it is implicitly translated to `<n`.
```

Comment:
The descriptions for `updated-others` and `updated-self` still refer to `updated`.

Also, no need to duplicate the examples, just say something like, 'Like `updated`, but only matches issues updated by other users/the current user'.